integration-tests/main-yaml/src/main/java/org/apache/camel/quarkus/main/GreetingBean.java

```diff
@@ -16,8 +16,13 @@
  */
 package org.apache.camel.quarkus.main;
 
+import javax.enterprise.context.ApplicationScoped;
```

Comment:
```suggestion
import jakarta.enterprise.context.ApplicationScoped;
```

Reply:
Oh yes indeed, that changed in main branch due to Quarkus 3 migration.

Reply:
Thanks for these suggestions. I changed these two imports as you proposed. Unfortunately, I can't build the main branch because of this error: Fatal error compiling: error: release version 17 not supported. So, I couldn't run the test locally before pushing.

Reply:
Camel 4 has Java 17 as minimum version. Would you happen in a way to use lower version ?

Note that [sdkman](https://sdkman.io/) is a tool that could help managing multiple java installed versions.

Reply:
I used java 11 because it's mentioned in the contributor's guide (https://github.com/apache/camel-quarkus/blob/main/docs/modules/ROOT/pages/contributor-guide/index.adoc). Java 17 fixed it for me, thanks!